src/react-to-webcomponent.ts

```diff
@@ -12,9 +12,9 @@ function toCamelCaseStyle(dashedStyle = "") {
   })
 }
 
-var define = {
+const define = {
   // Creates a getter/setter that re-renders everytime a property is set.
-  expando: function (receiver, key, value) {
+  expando: function (receiver: object, key: string, value: unknown) {
```

Comment:
Ideally, it's best to avoid `object` as it is so broadly defined. We _could_ use a generic here, but it would still likely require changing how we even use this define/expando object. Some advanced patterns in JS don't easily translate to using well-defined types in typescript.